src/utils.js

```diff
@@ -14,12 +17,20 @@ export function isWrappedModule(extensions) {
   }
 }
 
-export async function pathToRawSource(absPath) {
-  const fileURL = url.pathToFileURL(absPath);
+export async function pathToRawSource(p) {
+  const fileURL = url.pathToFileURL(p);
   return fs.promises.readFile(new URL(fileURL), 'utf8');
 }
 
-export function getSourceFromPathSync(absPath) {
-  const fileURL = url.pathToFileURL(absPath);
+export function getSourceFromPathSync(p) {
```

Comment:
It feels weird that it reads a file relative to the presm package root. Could calls to this be replaced with a direct call to `fs.readFileSync(p, 'utf8')`?

Reply:
Makes sense, I've updated this for now but suspect it will need further tweaking in #17